### PR TITLE
Feat: add lint frontend and lint all to Makefile.

### PR DESCRIPTION
**Pull request by iFurySt**
* style codes.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -147,10 +147,21 @@ install-precommit-hooks:
...
-lint:
+lint-backend:
 	@echo "$(YELLOW)Running linters...$(RESET)"
 	@poetry run pre-commit run --files opendevin/**/* agenthub/**/* --show-diff-on-failure --config $(PRECOMMIT_CONFIG_PATH)
 
+lint-frontend:
+	@echo "$(YELLOW)Running linters for frontend...$(RESET)"
+	@cd frontend && npm run lint
+
+lint-all:
```
Comment (li-boxuan) [inline Makefile:158]: nit: this feels redundant

Comment (iFurySt) [reply Makefile]: okay, have been deleted

File: Makefile [hunk 2]
```diff
@@ -147,10 +147,17 @@ install-precommit-hooks:
...
+lint-backend:
 	@echo "$(YELLOW)Running linters...$(RESET)"
 	@poetry run pre-commit run --files opendevin/**/* agenthub/**/* --show-diff-on-failure --config $(PRECOMMIT_CONFIG_PATH)
 
+lint-frontend:
+	@echo "$(YELLOW)Running linters for frontend...$(RESET)"
+	@cd frontend && npm run lint
+
+lint:
+	@$(MAKE) -s lint-all
```
Comment (rbren) [inline Makefile:159]: should this be `lint-frontend` and `lint-backend` now?

Comment (iFurySt) [reply Makefile:159]: oh, my fault.

File: frontend/package.json [hunk 3]
```diff
@@ -41,7 +41,7 @@
     "preview": "vite preview",
     "make-i18n": "node scripts/make-i18n-translations.cjs",
     "prelint": "npm run make-i18n",
-    "lint": "eslint src/**/*.ts* && prettier --check src/**/*.ts*",
+    "lint": "eslint src --ext .ts,.tsx,.js && prettier --check src/**/*.{ts,tsx}",
```
Comment (Sparkier) [inline frontend/package.json:44]: Somewhat unrelated but do we want a `lint-fix` as well?

**Review by Sparkier — APPROVED** [review]

**Comment by rbren** [comment]
@iFurySt looks like there's some FE lint

**Review by amanape — APPROVED** [review]

**Pull request merged**